Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/StorageProviderCommands/IndirectDataContainerLoadCommandTest.cs

```diff
@@ -67,7 +65,7 @@ public override void SetUp ()
     public void Initialization ()
     {
       Assert.That(_loadCommand.ObjectIDLoadCommand, Is.SameAs(_objectIDLoadCommandStub.Object));
-      Assert.That(_loadCommand.StorageProviderCommandFactory, Is.SameAs(_storageProviderFactoryStub.Object));
+      Assert.That(_loadCommand.RdbmsProviderCommandFactory, Is.SameAs(_storageProviderFactoryStub.Object));
     }
 
     [Test]
```

Comment:
please add unit test for the new overload

Reply:
I re-used the `_commandExecutionContextStub`, but through explicit cast to `IRdbmsProviderReadWriteCommandExecutionContext`, I can force the method call for the second overload.